test/test_ops.py

```diff
@@ -1706,6 +1706,16 @@ def map_to_fake(e):
 
                 res = op(sample.input, *sample.args, **sample.kwargs)
 
+                def outputs_alias_inputs(outputs, inputs):
+                    input_storages = set()
+                    for out in tree_flatten(outputs)[0]:
+                        if isinstance(out, torch.Tensor):
+                            input_storages.add(out.storage()._cdata)
+                    for inp in tree_flatten(inputs)[0]:
+                        if isinstance(inp, torch.Tensor) and inp.storage()._cdata in input_storages:
```

Comment:
should this use `StorageWeakRef(inp.storage())` instead of a private API 😛 (I guess it doesn't matter much since they do the same thing).